styles.css

```diff
@@ -454,17 +435,14 @@ p {
   justify-content: flex-end;
   align-items: center;
   margin-right: 3vw;
-
 }
 
 .card-header-background {
   display: flex;
   justify-content: space-between;
-
   width: 100%;
   height: 100%;
   background-color: #2B2B2B;
-
 }
 
 .arrow-button {
```

Comment:
These look good in your screenshots, but when I open it on my desktop after cloning down your repo it looks like this (if I can add screenshots in here, if not i will send you the image):

<img width="1143" alt="Screen Shot 2021-04-11 at 3 28 06 PM" src="https://user-images.githubusercontent.com/75296592/114321953-d9a5c580-9ada-11eb-8ee9-88dab69016ba.png">

Reply:
Same for me! In the dev tools, it looks like there are some 403 errors coming back on anything from Flaticon. Any time we load an image/icon from a link, we're subject to these sort of pitfalls, where something is blocked from loading or the host domain changes. To account for this, I'd start downloading all of your images/icons and adding them into your repo, and then importing them across your files!

![Screen Shot 2021-04-11 at 5 56 26 PM](https://user-images.githubusercontent.com/62263439/114325822-3f03b180-9aef-11eb-8b52-fe0d3a35e8c7.png)